pages/static/containers_js.go

```diff
@@ -28,7 +28,11 @@ function humanizeIEC(num) {
         var ret = humanize(num, 1024, ["TiB", "GiB", "MiB", "KiB", "Bytes"]);
 	return ret[0].toFixed(2) + " " + ret[1];
 }
-
+// IEC naming convention, with shorter abbreviations
```

Comment:
Wanna just change the one above to B and use it for both? I don't thinkwe mind :)